### ISSUE TITLE
After upgrading to 1.11.2 from 1.10 Dynmap no longer renders tiles

### ISSUE THREAD

**Issue by FrozenBeard** [webbukkit/dynmap#2085]
Startup
```
[10:10:46] [Server thread/INFO]: [dynmap] Loading dynmap v2.4-1148
[10:10:47] [Server thread/INFO]: [dynmap] version=git-Paper-1081 (MC: 1.11.2)
[10:10:47] [Server thread/INFO]: [dynmap] Unload queue not found - default to unload all chunks
[10:10:47] [Server thread/INFO]: [dynmap] Mod Support API available
[10:11:25] [Server thread/INFO]: [dynmap] Enabling dynmap v2.4-1148
[10:11:25] [Server thread/INFO]: [dynmap] Using Bukkit Permissions (superperms) for access control
[10:11:25] [Server thread/INFO]: [dynmap] Web interface permissions only available for online users
[10:11:28] [Server thread/INFO]: [dynmap] Mod Support processing completed
[10:11:29] [Server thread/INFO]: [dynmap] Loaded 24 shaders.
[10:11:29] [Server thread/INFO]: [dynmap] Loaded 82 perspectives.
[10:11:29] [Server thread/INFO]: [dynmap] Loaded 14 lightings.
[10:11:30] [Server thread/INFO]: [dynmap] version 2.4-1148 is enabled - core version 2.4-2082
[10:11:30] [Server thread/INFO]: [dynmap] Loaded 2 maps of world 'PirateCraft_1'.
[10:11:30] [Server thread/INFO]: [dynmap] Loaded 19535 pending tile renders for world 'PirateCraft_1'
[10:11:30] [Server thread/INFO]: [dynmap] Loaded 2 maps of world 'PirateCraft_1_nether'.
[10:11:31] [Server thread/INFO]: [dynmap] Loaded 679 pending tile renders for world 'PirateCraft_1_nether'
[10:11:31] [Server thread/INFO]: [dynmap] Loaded 2 maps of world 'PirateCraft_1_the_end'.
[10:11:31] [Server thread/INFO]: [dynmap] Loaded 266 pending tile renders for world 'PirateCraft_1_the_end'
[10:11:31] [Server thread/INFO]: [dynmap] Enabled
```

When I run a radius render to test it:
```
[10:13:11] [Dynmap Render Thread/INFO]: [dynmap] Loading resource pack dokucraft.zip

[10:13:21] [Dynmap Render Thread/ERROR]: [dynmap] Exception during render job: world=PirateCraft_1, map=null
[10:13:21] [Dynmap Render Thread/WARN]: java.lang.ArrayIndexOutOfBoundsException: 22528
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.combineSubimageFromImage(TexturePack.java:896)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.makeShulkerSideImage(TexturePack.java:1076)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.patchShulkerImages(TexturePack.java:1104)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.processDynamicImage(TexturePack.java:1344)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.<init>(TexturePack.java:852)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.getTexturePack(TexturePack.java:683)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePackHDShader.getTexturePack(TexturePackHDShader.java:57)
[10:13:21] [Dynmap Render Thread/INFO]: [dynmap] Loading resource pack dokucraft.zip
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePackHDShader.access$000(TexturePackHDShader.java:22)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePackHDShader$ShaderState.<init>(TexturePackHDShader.java:128)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePackHDShader.getStateInstance(TexturePackHDShader.java:323)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.HDMapManager.getShaderStateForTile(HDMapManager.java:136)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.IsoHDPerspective.render(IsoHDPerspective.java:1088)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.HDMapTile.render(HDMapTile.java:79)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.MapManager$FullWorldRenderState.processTile(MapManager.java:700)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.MapManager$FullWorldRenderState.run(MapManager.java:641)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$1.run(MapManager.java:190)
[10:13:21] [Dynmap Render Thread/WARN]: 	at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$2.run(MapManager.java:206)
[10:13:21] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[10:13:21] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[10:13:21] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
[10:13:21] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
[10:13:21] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[10:13:21] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[10:13:21] [Dynmap Render Thread/WARN]: 	at java.lang.Thread.run(Thread.java:745)
[10:13:22] [Dynmap Render Thread/ERROR]: [dynmap] Exception during render job: world=PirateCraft_1, map=null
[10:13:22] [Dynmap Render Thread/WARN]: java.lang.ArrayIndexOutOfBoundsException: 22528
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.combineSubimageFromImage(TexturePack.java:896)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.makeShulkerSideImage(TexturePack.java:1076)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.patchShulkerImages(TexturePack.java:1104)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.processDynamicImage(TexturePack.java:1344)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.<init>(TexturePack.java:852)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePack.getTexturePack(TexturePack.java:683)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePackHDShader.getTexturePack(TexturePackHDShader.java:57)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePackHDShader.access$000(TexturePackHDShader.java:22)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePackHDShader$ShaderState.<init>(TexturePackHDShader.java:128)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.TexturePackHDShader.getStateInstance(TexturePackHDShader.java:323)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.HDMapManager.getShaderStateForTile(HDMapManager.java:136)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.IsoHDPerspective.render(IsoHDPerspective.java:1088)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.hdmap.HDMapTile.render(HDMapTile.java:79)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.MapManager$FullWorldRenderState.processTile(MapManager.java:700)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.MapManager$FullWorldRenderState.run(MapManager.java:641)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$1.run(MapManager.java:190)
[10:13:22] [Dynmap Render Thread/WARN]: 	at org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$2.run(MapManager.java:206)
[10:13:22] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[10:13:22] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[10:13:22] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
[10:13:22] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
[10:13:22] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[10:13:22] [Dynmap Render Thread/WARN]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[10:13:22] [Dynmap Render Thread/WARN]: 	at java.lang.Thread.run(Thread.java:745)
```

**Comment by SimonOrJ** [comment]
Duplicate of #2067

**Comment by FedUpWith-Tech** [comment]
fixed in #2109
